### ISSUE TITLE
The <div id="description> element Description blocks' content in HTML has become more narrow

### ISSUE THREAD

**Issue by FArekkusu** [codewars/codewars.com#2352]
The content of the description blocks in HTML has become narrow.

![image](https://user-images.githubusercontent.com/31668662/113509609-35ae9e00-955f-11eb-9624-b2b98fa756a5.png)

![image](https://user-images.githubusercontent.com/31668662/113509806-2d0a9780-9560-11eb-9930-bfa8a965aba5.png)

**Browser**

- OS: Windows 7
- Browser: Google Chrome
- Version: 89.0.4389.114

**Comment by Blind4Basics** [comment]
(I actually like the narrower thing. On my screen, that's rather weird to have the full width used and that doesn't help readability. If this narrowing doesn't happen on small screens, it might actually be a good idea)

**Comment by kazk** [comment]
Making it narrow was intentional. The description becomes difficult to read on wider screen. But it does look broken. I'll make the whole page narrow, similar to the dashboard and profile.

**Comment by kazk** [comment]
> If this narrowing doesn't happen on small screens

It doesn't. It's a max width.

**Comment by Blind4Basics** [comment]
Maybe the same should be applied in the solutions page and the discourse? (`show kata description`) For now it's a bit weird that the details page is the only one showing the description like this.
(Next question would be: what about the trainer?)

**Comment by kazk** [comment]
> Maybe the same should be applied in the solutions page and the discourse?

Yeah, I was lazy, and didn't think about those. I ran into a super ugly description because of the width combined with centered formulas/tables, and added this constraint without thinking much.

> (Next question would be: what about the trainer?)

I wasn't thinking of the trainer because it doesn't get as wide.

**Comment by Voileexperiments** [comment]
I don't think we really *need* to limit max-width at all. Limiting max-width are for reading *passages* where a paragraph is long and it can be disorienting trying to find the next line on the other side; most CW katas have paragraphs so short they seldom spans more than one line.

Also, currently this is set to a flat `64em` (which is `1024px`) which is a very horrible implementation: it might look good in 1366x768 but anything bigger would make it look very inefficient, which includes 1920x1080. I believe most implementations either assigns different values for each breakpoint, or set it to a higher value (around 1100-1600px).
